src/pages/view-test-result/cat-d/components/debrief-card/__tests__/debrief-card.spec.ts

```diff
@@ -0,0 +1,344 @@
+import { configureTestSuite } from 'ng-bullet';
+import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
+import { CatDUniqueTypes } from '@dvsa/mes-test-schema/categories/D';
+import { CatD1UniqueTypes } from '@dvsa/mes-test-schema/categories/D1';
+import { CatDEUniqueTypes } from '@dvsa/mes-test-schema/categories/DE';
+import { CatD1EUniqueTypes } from '@dvsa/mes-test-schema/categories/D1E';
+
+describe('DebriefCardComponent', () => {
+  let fixture: ComponentFixture<DebriefCardComponent>;
+  let component: DebriefCardComponent;
+  let faultSummaryProvider: FaultSummaryProvider;
+  let faultCountProvider: FaultCountProvider;
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DebriefCardComponent,
+        MockComponent(TickIndicatorComponent),
+        MockComponent(DangerousFaultBadgeComponent),
+        MockComponent(SeriousFaultBadgeComponent),
+        MockComponent(DrivingFaultsBadgeComponent),
+        MockComponent(DataRowComponent),
+        MockComponent(DataRowCustomComponent),
+        MockComponent(DataRowWithListComponent),
+        MockComponent(FaultsDataRowComponent),
+        MockComponent(VehicleChecksDataRowComponent),
+      ],
+      imports: [
+        IonicModule,
+      ],
+      providers: [
+        { provide: Config, useFactory: () => ConfigMock.instance() },
+        FaultSummaryProvider,
+        FaultCountProvider,
+      ],
+    });
+  });
+
+  beforeEach(async(() => {
+    fixture = TestBed.createComponent(DebriefCardComponent);
+    component = fixture.componentInstance;
+    faultSummaryProvider = TestBed.get(FaultSummaryProvider);
+    faultCountProvider = TestBed.get(FaultCountProvider);
+  }));
+
+  describe('Class', () => {
+    describe('getTestRequirements', () => {
+      it('should return the correct values for all test requirements for Cat D', () => {
+        const data: CatDUniqueTypes.TestData = {
+          testRequirements: {
+            angledStartControlledStop: true,
+            downhillStart: false,
+            normalStart1: true,
+            normalStart2: false,
+            busStop1: true,
+            busStop2: true,
+          },
+        };
+        component.data = data;
+        component.category = TestCategory.D;
+        fixture.detectChanges();
+        const result: DataRowListItem[] = component.getTestRequirements();
+
+        expect(result.length).toEqual(7);
+        expect(result).toContain({ label: TestRequirementsLabels.normalStart1, checked: true });
+        expect(result).toContain({ label: TestRequirementsLabels.normalStart2, checked: false });
+        expect(result).toContain({ label: TestRequirementsLabels.uphillStart, checked: false });
+        expect(result).toContain({ label: TestRequirementsLabels.downhillStart, checked: false });
+        expect(result).toContain({ label: TestRequirementsLabels.angledStartControlledStop, checked: true });
+      });
```

Comment:
would be nice just to check that the busStop1 and 2 come back populated properly like the ones currently there. (this applies to tests below)